packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -190,20 +191,43 @@ class _ModalBottomSheet<T> extends StatefulWidget {
 class _ModalBottomSheetState<T> extends State<_ModalBottomSheet<T>> {
   @override
   Widget build(BuildContext context) {
+    final MediaQueryData mediaQuery = MediaQuery.of(context);
+    final MaterialLocalizations localizations = MaterialLocalizations.of(context);
+    String routeLabel;
+    switch (defaultTargetPlatform) {
+      case TargetPlatform.iOS:
+        routeLabel = '';
+        break;
+      case TargetPlatform.android:
+      case TargetPlatform.fuchsia:
+        routeLabel = localizations.dialogLabel;
+        break;
+    }
+
     return new GestureDetector(
+      excludeFromSemantics: true,
```

Comment:
I'm not exactly sure if we should exclude this from semantics, since we have the dismiss barrier on iOS and the back button for Android.